src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -98,6 +117,125 @@ public void should_return_a_200_and_retrieve_a_task_by_event_and_case_match() {
         common.cleanUpTask(taskId, REASON_COMPLETED);
     }
 
+    @Test
+    public void should_return_a_200_and_retrieve_single_task_by_event_and_case_match_and_assignee() {
+        final String assigneeId = getAssigneeId(authenticationHeaders);
+
+        // create a caseId
+        final String caseId = given.iCreateACcdCase();
```

Comment:
the line of code is self-explanatory, do we need the comment?

Reply:
I am working on other related ticket, so will make changes.